docs/source/item-validation.rst

```diff
@@ -193,6 +221,36 @@ as a `dict`:
         OtherItem: '/path/to/otheritem_schema.json',
     }
 
+.. _SPIDERMON_VALIDATION_CERBERUS:
+
+SPIDERMON_VALIDATION_CERBERUS
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Default: ``None``
+
+A `list` containing the local path of the item schema.
```

Comment:
This does not match the example below, which in addition to a local path includes a URL and a schema defined as a  Python dictionary.

Reply:
Missed that bit, the support for URL's was added a wee bit later. Fixed now.